libslide/src/bignum/add.rs

```diff
@@ -7,7 +8,14 @@ impl ops::Add for Bignum {
     type Output = Bignum;
 
     fn add(self, rhs: Bignum) -> Bignum {
-        let is_neg: bool = rhs.is_neg && self.is_neg;
+        let mut is_neg: bool = false;
+        if self.is_neg && rhs.is_neg {
+            is_neg = true;
```

Comment:
```suggestion
        let mut is_neg: bool = self.is_neg && rhs.is_neg;
        if self.is_neg && rhs.is_neg {
            is_neg = true;
```

And just get rid of this branch